apps/src/lib/ui/accounts/ManageLinkedAccounts.jsx

```diff
@@ -71,6 +75,14 @@ class ManageLinkedAccounts extends React.Component {
     return true;
   };
 
+  shouldDisableConnectButton = authOption => {
+    return (
+      PERSONAL_LOGIN_TYPES.includes(authOption.credentialType) &&
+      !this.formatEmail(authOption) &&
```

Comment:
Why is `!this.formatEmail(authOption)` included in the conditions for disabling the buttons?

Reply:
Good question. When there's an id/email, then it is already connected and it shouldn't disable the button because it's then a disconnect button. The easier way to express this is just `!authOption.id` so I've updated it to that. I've also added a comment breaking down the conditional and addressing the logic.

Reply:
Ahh ok, thanks for the explanation. That makes sense now, and I think doing `!authOption.id` is much easier to grok.